menuinst/platforms/base.py

```diff
@@ -75,7 +75,7 @@ def placeholders(self) -> Dict[str, str]:
             "DISTRIBUTION_NAME": self.base_prefix.name,
             "BASE_PYTHON": str(self.base_prefix / "bin" / "python"),
             "PREFIX": str(self.prefix),
-            "ENV_NAME": self.prefix.name,
+            "ENV_NAME": self.env_name or "",
```

Comment:
Can this be false-y? 🤔 I think we don't need the `or ""` part, right?

Reply:
Now that I look at it again, I don't think so. I guess I was overly cautious here.